test/DataTable/__snapshots__/DataTable.test.tsx.snap

```diff
@@ -20,7 +20,7 @@ exports[`DataTable renders correctly 1`] = `
       <tr>
         <th
           colspan="1"
-          style="cursor: pointer;"
+          style="cursor: pointer; opacity: 1; position: relative; width: 150px; z-index: 0;"
```

Comment:
@skuengneo I have the impression something is not correct with the style. Is correct that this is always applied?

Reply:
@manni497 I changed it so the styles should only be applied if there is a tleast 1 pinned column. When testing with f.e. the bridgestone solution it looks right. Not sure why this is happening here in the snapshot..